packages/attribute-registry-process/src/services/readModelService.ts

```diff
@@ -171,7 +171,7 @@ export function readModelServiceBuilder(
       const nameFilter = name
         ? {
             "data.name": {
-              $regex: name,
+              $regex: ReadModelRepository.escapeRegExp(name),
```

Comment:
```suggestion
              $regex: new RegExp(ReadModelRepository.escapeRegExp(name)),
```
Shouldn't we add `new RegExp()` in all the regexes? If they were missing now could be a good opportunity to add them.